src/main/java/com/marginallyclever/makelangelo/Makelangelo.java

```diff
@@ -1,6 +1,6 @@
 package com.marginallyclever.makelangelo;
-/**
```

Comment:
why?

Reply:
good question, I'll revert this modification